src/main/java/step2/domain/Rank.java

```diff
@@ -0,0 +1,37 @@
+package step2.domain;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public enum Rank {
+    FIRST(6, 2000000000),
```

Comment:
돈과 같은 1000단위로 구분은 다음과 같이 underbar(_)를 활용하면 코드의 가독성이 좋아질 수 있어요!
```suggestion
    FIRST(6, 2_000_000_000),
```

Reply:
꿀팁 감사합니다